.github/workflows/ci-test.yml

```
@@ -0,0 +1,50 @@
+
+      # set up JDK 17
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      # Set up Gradle cache
+      - name: Cache Gradle package
+        uses: actions/cache@v4
+        with: 
```

yamllint

[error] trailing spaces

(trailing-spaces)